Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "RAI"
 uuid = "9c30249a-7e08-11ec-0e99-a323e937e79f"
-version = "0.2.7"
+version = "0.2.7-alpha"
```

Comment:
0.2.7 is already released, so I think this would be 0.2.8-alpha, right?

@quinnj: Do julia packages support `-alpha` versions?

Reply:
I don't think they do as far as I know